flipperdevices/flipperzero-firmware
# Services: simplify api

# What's new

- Dolphin: replace macros with function

# Verification 

- Check firmware size
- Check that Dolphin works correctly

# Checklist (For Reviewer)
